intro-dplyr.Rmd

```diff
@@ -0,0 +1,343 @@
+---
+title: "Work with and manage your data efficiently in R: dplyr and tidyr"
+params:
+    datetime: "2018-10-26"
+    level: "Beginner" # or intermediate or advanced
+    videolink: "" # Keep empty if no link yet.
+    instructor: "Elena Dudukina"
+---
+
+```{r setup, echo=FALSE}
+source("R/utils.R")
+```
+
+## Session details
+
+- **Date of session**: `r format_date(params$datetime)`
+- **Instructor**: `r params$instructor`
+- **Session level**: `r params$level`
+    - **Part of the ["Beginner Series"](index.html)**
+- **Required packages to install**:
+
+    ```{r, eval=FALSE}
+    install.packages(c("dplyr", "tidyr"))
```

Comment:
I changed this to be just dplyr and tidyr. I'd rather have the sessions be self contained and focused as much as possible